pennylane/data/attributes/operator/operator.py

```diff
@@ -25,6 +25,7 @@
 from pennylane.data.base.attribute import DatasetAttribute
 from pennylane.data.base.hdf5 import HDF5Group, h5py
 from pennylane.operation import Operator, Tensor
+from pennylane.ops.qubit.hamiltonian import Hamiltonian as LegacyHamiltonian
```

Comment:
Glad this works 👍 Alternatively could do
```
with qml.operation.disable_new_opmath_cm():
    LegacyHamiltonian = qml.Hamiltonian
```

Reply:
Yes I think the way you are doing it is nicer and cleaner!

Reply:
Just wasnt sure this would actually work ^^

Reply:
I think it depends on where in the codebase you are doing this because
```
import pennylane as qml
from pennylane.ops.qubit import Hamiltonian

Hamiltonian
```

```
pennylane.ops.op_math.linear_combination.LinearCombination
```

Reply:
Yep, I think the difference is in the import order. We complete all the imports before calling `enable_new_opmath` in the `__init__`, so the expected behaviour is retained. When you do it outside while using the PennyLane as a module, you will see this behaviour.